packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -89,140 +103,131 @@ const onKeyDown: React.KeyboardEventHandler = e => {
-    const content = (
-      <OrganizationOption
-        orgName={datum.orgName}
-        planType={datum.planType}
-        href={props.constructOrganizationURL(datum.orgId)}
-      />
-    );
+  const errorMessage =
+    'You do not belong to any organizations. Create an organization to start using MongoDB Cloud';
+
+  return (
+    <Menu
+      trigger={
+        <OrganizationTrigger
+          current={current?.orgName ?? 'All Organizations'}
+          className={className}
+          urls={urls}
+          isActive={isActive}
+        />
+      }
+      className={menuContainerStyle}
+      justify="start"
+    >
+      <FocusableMenuItem>
+        <Input
+          onChange={onChange}
+          onKeyDown={onKeyDown}
+          variant="organization"
+        />
+      </FocusableMenuItem>
+
+      <>{data?.map(renderOrganizationOption) ?? <li>{errorMessage}</li>}</>
+
+      <MenuSeparator />
+      <MenuItem
+        onKeyDown={onKeyDown}
+        href={urls.mongoSelect?.viewAllOrganizations}
+      >
+        <strong className={viewAllStyle}>View All Organizations</strong>
+      </MenuItem>
+    </Menu>
+  );
+}
+
+OrgSelect.displayName = 'OrgSelect';
+
+export { OrgSelect };
+
+function ProjectSelect({
+  current,
+  className,
+  onChange,
+  data,
+  onClick,
+  constructProjectURL,
+  urls,
+}: ProjectMongoSelectProps) {
+  const renderProjectOption = (datum: ProjectInterface) => {
+    const { projectId, projectName, orgId } = datum;
 
     return (
       <MenuItem
-        key={datum.orgId}
-        className={menuItemContainerStyle}
+        key={projectId}
+        className={cx(menuItemContainerStyle, nameStyle)}
         onClick={onClick}
+        href={constructProjectURL(orgId, projectId)}
       >
-        {content}
+        {projectName}
       </MenuItem>
     );
   };
 
   return (
-    <Menu trigger={trigger} className={menuContainerStyle} justify="start">
+    <Menu
+      trigger={
+        <ProjectTrigger current={current.projectName} className={className} />
+      }
+      className={menuContainerStyle}
+      justify="start"
+    >
       <FocusableMenuItem>
-        <Input onChange={onChange} onKeyDown={onKeyDown} variant={variant} />
+        <Input onChange={onChange} onKeyDown={onKeyDown} variant="project" />
       </FocusableMenuItem>
 
-      <li role="none" className={listContainerStyle}>
-        <ul
-          onKeyDown={e => e.preventDefault()}
-          className={ulContainerStyle}
-          role="menu"
-        >
-          {isProject(props)
-            ? props.data.map(datum => renderProjectOption(props, datum))
-            : props.data.map(datum => renderOrganizationOption(props, datum))}
-        </ul>
-      </li>
+      <>{data?.map(datum => renderProjectOption(datum))}</>
 
       <MenuSeparator />
 
-      {footer}
+      <li onKeyDown={onKeyDown} role="none" className={projectButtonStyle}>
+        <FocusableMenuItem>
+          <Button href={urls.mongoSelect.viewAllProjects} as="a">
```

Comment:
You're using href, you don't need to specify `as="a"`. Perhaps that default behavior is confusing?

Reply:
Was supplying as "a" because typescript was typing urls.mongoSelect.newProject as possibly undefined, for now typecasting as string but need to figure out why Typescript is confused about that because should be <Required> by the time it gets to MongoSelect